### PR TITLE
Amended the final readme file and deployed the page

### PR DESCRIPTION
**Pull request by andrianarivo**
# In this final PR, we: 

- Used JavaScript to make websites dynamic and build basic single page apps
- Used ES6 syntax
- Used ES6 modules
- Used callbacks and promises
- Used webpack
- Applied JavaScript best practices and preprocessor scripting language guides in code
- Used AAA pattern for unit tests
- Wrote units tests for a JavaScript app
- Followed Gitflow
- Solved simple git conflicts
- Sent and received data from an API
- Used API documentation
- Understood and use JSON
- Made JavaScript code asynchronous
- Performed multiple code review for a team member

### CODE REVIEW

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi Team
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 1]
```diff
@@ -38,25 +38,33 @@
...
           <p id="modal-summary">SUMMARY</p>
-          <span id="comments-counter"
-            >COMMENTS (2)
+          <span
+            >COMMENTS <strong><span id="comments-counter">2</span></strong>
             <table>
               <tbody id="table-body">
-                <!--fetch the comments-->
+                <tr>
+                  <!--fetch the comments-->
```
[the remark this answers is not in the document]
Comment (andrianarivo) [reply dist/index.html]: @joel-kalema we can delete the LocalStorage, no problem, but are you sure we are not fetching and updating the data in the API ? please re-consider this before we do updates about it.

**Review by bimbolabuari — COMMENTED** [review]
Hello @joel-kalema  👋,

_This is an additional review as a result of the second opinion request._

- Kudos on working on this milestone 👍 
  - As required we need to use the [involvement API](https://microverse.notion.site/Involvement-API-869e60b5ad104603aa6db59e08150270) to post user interactions. 
    -   For Parameters for POST action:
There should be a unique identifier of the app i.e the app_id as a URL parameter and also a request body with a JSON object with the following keys: `item_id`, `username`, `comment` (mandatory)

- You should use the API to send and receive data and not the local storage, I hope this is clearer 👍 


Keep rocking 🚀! 

@joel-kalema, the PR does not follow the correct Github flow as required and should have invalidated, let's take note of this moving forward 👍

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented 🥇   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**